sources/packages/backend/apps/db-migrations/src/sql/Reports/Update-data-inventory-report.sql

```diff
@@ -0,0 +1,527 @@
+            inner join sims.program_years program_years on applications.program_year_id = program_years.id
+            left join sims.cra_income_verifications cra_income_verifications on applications.id = cra_income_verifications.application_id
+            and cra_income_verifications.supporting_user_id is null
+            left join (
+                select
+                    *
+                from
+                    sims.supporting_users supporting_users
+                where
+                    supporting_users.supporting_user_type = ''Parent''
+                order by
+                    supporting_users.created_at asc
+                limit
+                    1
+            ) parent1 on applications.id = parent1.application_id
+            left join sims.users parent1_users on parent1.user_id = parent1_users.id
+            left join (
+                select
+                    *
+                from
+                    sims.supporting_users supporting_users
+                where
+                    supporting_users.supporting_user_type = ''Parent''
+                order by
+                    supporting_users.created_at desc
+                limit
+                    1
+            ) parent2 on applications.id = parent2.application_id
+            left join sims.users parent2_users on parent2.user_id = parent2_users.id
+            left join sims.supporting_users partner on partner.supporting_user_type = ''Partner''
+            and applications.id = partner.application_id
+            left join sims.users partner_users on partner.user_id = partner_users.id
+            left join sims.student_assessments current_assessment on applications.current_assessment_id = current_assessment.id
+            left join sims.institution_locations institution_locations on institution_locations.id = applications.location_id
+            left join sims.institutions institutions on institutions.id = institution_locations.institution_id
+			      left join sims.institution_type institution_types on institution_types.id = institutions.institution_type_id
+            left join sims.education_programs education_programs on education_programs.id = applications.pir_education_program_id
+            left join sims.education_programs_offerings education_programs_offerings on current_assessment.offering_id = education_programs_offerings.id
+            left join sims.student_scholastic_standings student_scholastic_standings on applications.id = student_scholastic_standings.application_id
+            left join (
+            	select sr.student_id, ''Yes'' as current_restriction
+            	from sims.student_restrictions sr
+            	where sr.is_active = true
+            	limit 1
+            ) as current_student_restrictions on current_student_restrictions.student_id = students.id
+            left join (
+            	select sr.student_id, ''Yes'' as previous_restriction
```

Comment:
For my understanding, there was some performance benefit using the `left join` instead of a `select exists` in the main `select` of the query, is it right?

Reply:
Not a performance benefit. It could be a "select exists" if it were in the select part but as we are moving from subqueries in the select part to left joins, that's the way I found to do the same in a join.